Lib/test/test_unpack.py

```diff
@@ -135,13 +144,57 @@
       ...
     TypeError: cannot unpack non-iterable int object
 
-Unpacking to an empty iterable should raise ValueError
+Unpacking to an empty iterable should raise UnpackError
 
     >>> () = [42]
     Traceback (most recent call last):
       ...
-    ValueError: too many values to unpack (expected 0)
+    UnpackError: too many values to unpack (expected 0, got 1)
+
+Unpacking to an empty iterable should raise UnpackError, but it won't consume the
```

Comment:
It does consume it though. I mean, it consumes as many values as it is expecting, right?

Reply:
Yes, but the comment says otherwise. In addition, it is about "an empty iterable". I just wanted to highlight that the description is not what the test is doing.

Reply:
Bad copy-paste job on my end! I'll fix this line entirely.